crl-test/outputs.tf

```
@@ -4,6 +4,11 @@
 # output service accounts.
 #
 
-output "service_account_email" {
-  value = google_service_account.crl.email
+output "service_account_email_admin" {
+  value = google_service_account.crl-admin.email
```

Please make all Terraform resource names use underscores instead of dashes.
```suggestion
  value = google_service_account.crl_admin.email
```